tool/gravity/cli/operation.go

```diff
@@ -508,7 +510,8 @@ func (r *backendOperations) listUpdateOperation(environ LocalEnvironmentFactory)
 }
 
 func (r *backendOperations) listJoinOperation(environ LocalEnvironmentFactory) error {
-	env, err := environ.NewJoinEnv()
+	// TODO(dmitri): assumes local state inside process's working directory
```

Comment:
What does this mean? Join-related plan commands have to be run from a specific directory? Is that a change in behavior?

Reply:
No, this is not a change in behavior - it is note to self when we need to change this behavior.